shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/OCert.hs

```diff
@@ -129,3 +138,18 @@ kesPeriod (SlotNo s) =
     if spkp == 0
       then error "kesPeriod: slots per KES period was set to zero"
       else KESPeriod . fromIntegral $ s `div` spkp
+
+-- | Signable part of an operational certificate
+data OCertSignable crypto
+  = OCertSignable !(VerKeyKES crypto) !Natural !KESPeriod
+
+instance Crypto crypto => SignableRepresentation (OCertSignable crypto) where
+  getSignableRepresentation (OCertSignable vk counter period) =
+    KES.rawSerialiseVerKeyKES vk
+      <> BSL.toStrict (Binary.encode counter)
+      <> writeBinaryWord64 (fromIntegral $ unKESPeriod period)
```

Comment:
These two should both be at-most 64bit numbers, so we should serialise them the same.

tbh, we should probably use Word64 for the cert counter not natural.